### ISSUE TITLE
G183 Example 2 doesn't pass the G183 test procedure

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#2243]
Example 2 describes text that has a 3.9:1 contrast with the surrounding text. It does not mention focus or hover style differences.

> The hypertext links in a document are medium-light blue (#3366CC) and the regular text is black (#000000). Because the blue text is light enough, it has a contrast of 3.9:1 with the surrounding text and can be identified as being different than the surrounding text by people with all types of color blindness, including those individuals who cannot see color at all.

https://www.w3.org/WAI/WCAG21/Techniques/general/G183#example-2

This appears to fail steps 3 and 4 of the test procedure:

1) Locate all instances where color alone is used to convey information about text.
2) Check that the [relative luminance](https://www.w3.org/TR/WCAG21/#) of the color of the text differs from the relative luminance of the surrounding text by a contrast ratio of at least 3:1.
3) Check that pointing (mouseover) to the link causes a visual enhancement (such as an underline, font change, etc.)
4) Check that moving keyboard focus to the link causes a visual enhancement (such as an underline, font change, etc.)

Expected Results
1) Checks 2, 3, and 4 are all true.

Related: https://github.com/w3c/wcag/issues/201

**Comment by patrickhlauke** [comment]
https://github.com/w3c/wcag/pull/2244
